sass/README.md

```diff
@@ -0,0 +1,41 @@
+SASS Style Guide
+=====
+### Formatting
+* Use the SASS syntax.
```

Comment:
That's a topic for a different PR, but in general, I'm more inclined for using the newer syntax: Scss